public/components/common/query_utils/index.ts

```diff
@@ -205,19 +217,28 @@ export const getIndexPatternFromRawQuery = (query: string): string => {
   return getPromQLIndex(query) || getPPLIndex(query);
 };
 
+function extractSpanAndResolution(query: string) {
+  const regex = /'(\d+)([smhdwMy])'/;
+  const match = query.match(regex);
+  // eslint-disable-next-line radix
+  return match ? { span: parseInt(match[1]), resolution: match[2] } : null;
+}
+
 export const preprocessMetricQuery = ({ metaData, startTime, endTime }) => {
   // convert to moment
   const start = convertDateTime(startTime, true);
   const end = convertDateTime(endTime, false);
 
-  const resolution = findMinInterval(start, end);
+  // const resolution = findMinInterval(start, end);
+  const spanResolution = extractSpanAndResolution(metaData.query);
+  console.log('blah: ', spanResolution);
```

Comment:
There are a lot of console.log statements in this PR can we please remove all of them?

Reply:
removed all of console log comments